src/hooks/useModal.tsx

```diff
@@ -0,0 +1,30 @@
+import { useSetRecoilState } from "recoil";
+
+import modalList from "@recoil/modalList";
+import { ModalType } from "@core/types/modal";
+
+const useModal = () => {
+  const setModalList = useSetRecoilState(modalList);
+
+  const showModal = (modalName: ModalType) => {
+    setModalList((prev) => {
+      if (prev.length > 1) {
+        return [...prev, modalName];
+      } else {
+        return [modalName];
+      }
+    });
```

Comment:
```suggestion
    setModalList((prev) => {
      if (prev.length > 1) return [...prev, modalName];
      else return [modalName];
    });
```
이 부분에 중괄호가 조큼 많아서 개인적으로는
if, else if, else 안의 실행해야 하는 코드가 한 줄이라면 위와 같이 선호하는 편인데 어떠신가요~~~?

Reply:
이건 정말 취향 차인데 요즘에는 중괄호 치는게 좋아서 그냥 쳤어요 ㅋㅋ
뭔가 한줄 한줄 따라가면서 보기 좋아서

Reply:
말 나온김에 그냥 저렇게 삼항으로 바꿔야겠다